beta/src/components/MDX/Challenges/Challenges.tsx

```diff
@@ -75,13 +75,15 @@ export function Challenges({
   const challenges = parseChallengeContents(children);
   const totalChallenges = challenges.length;
   const scrollAnchorRef = React.useRef<HTMLDivElement>(null);
+  const [activeChallengeHasChanged, setActiveChallengeHasChanged] =
+    React.useState(false);
```

Comment:
Add `activeChallengeHasChanged` to flag whether the activeChallenge has changed to avoid scrolling to Challenges when page mounted.